src/test/java/net/minecraftforge/debug/misc/OnAnvilUpdateEventFix.java

```diff
@@ -0,0 +1,29 @@
+package net.minecraftforge.debug.misc;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import net.minecraftforge.event.AnvilUpdateEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+
+@Mod(OnAnvilUpdateEventFix.MOD_ID)
+@Mod.EventBusSubscriber(modid = OnAnvilUpdateEventFix.MOD_ID)
```

Comment:
Instead of `EventBusSubscriber`, register the event listener in the mod constructor, blocked behind the `ENABLED` flag.